policy-report/pkg/api/wgpolicyk8s.io/v1beta1/policyreport_types.go

```diff
@@ -21,6 +21,24 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// +kubebuilder:validation:Enum=pass;fail;warn;error;skip
```

Comment:
Add comment:
`StatusFilter is used by PolicyReport generators to write only those reports whose status is specified by the filters`